pkg/routeagent/controllers/route/route.go

```diff
@@ -458,6 +472,7 @@ func (r *Controller) processNextEndpoint() bool {
 
 		r.gwVxLanMutex.Lock()
 		r.isGatewayNode = false
+		klog.Infof("Creating the vxlan interface: %s (non-gw)", VxLANIface)
```

Comment:
```suggestion
		klog.Infof("Creating the vxlan interface %s with gateway node IP %s", VxLANIface, localClusterGwNodeIP)
```

More context can't hurt here.

Reply:
Done.